### PR TITLE
Add curved footer image to publication and news pages

### LINKED ISSUE #250: Add curved footer image to publication and news pages
The publication and news pages have a curved footer image.

Note: The coveo search uses the same footer, refactor.

Test URLs:

Publication
- https://www.moleculardevices.com/newsroom/in-the-news
- https://www.moleculardevices.com/newsroom/in-the-news/overcoming-analysis-paralysis-in-cell-painting-with-artificial-intelligencediscovery

News
- https://www.moleculardevices.com/newsroom/news
- https://www.moleculardevices.com/newsroom/news/advanced-solutions-life-sciences-collaborate-to-develop-3d-biology-automation-technologies-for-drug-discovery

Coveo Search
- https://www.moleculardevices.com/search-results

### PR DESCRIPTION
**Pull request by lydiapuric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #250

Test URLs:

Before: https://main--moleculardevices--hlxsites.hlx.page/newsroom/news
After: https://curved-footer--moleculardevices--hlxsites.hlx.page/newsroom/news

AND

Before: https://main--moleculardevices--hlxsites.hlx.page/newsroom/in-the-news
After: https://curved-footer--moleculardevices--hlxsites.hlx.page/newsroom/in-the-news

AND

Before: https://main--moleculardevices--hlxsites.hlx.page/newsroom/in-the-news/overcoming-analysis-paralysis-in-cell-painting-with-artificial-intelligencediscovery
After: https://curved-footer--moleculardevices--hlxsites.hlx.page/newsroom/in-the-news/overcoming-analysis-paralysis-in-cell-painting-with-artificial-intelligencediscovery

AND

Before: https://main--moleculardevices--hlxsites.hlx.page/newsroom/news/advanced-solutions-life-sciences-collaborate-to-develop-3d-biology-automation-technologies-for-drug-discovery
After: https://curved-footer--moleculardevices--hlxsites.hlx.page/newsroom/news/advanced-solutions-life-sciences-collaborate-to-develop-3d-biology-automation-technologies-for-drug-discovery

AND

Before: https://main--moleculardevices--hlxsites.hlx.page/search-results
After: https://curved-footer--moleculardevices--hlxsites.hlx.page/search-results

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -989,4 +989,20 @@ main .section[aria-hidden="true"] {
   left: 0;
   right: 0;
   z-index: 999;
+}
+
+.section.background-grey.coveo-search-container::after,
```
Comment (mhaack) [inline styles/styles.css:994]: `background-grey` can be remove here

Comment (lydiapuric) [reply styles/styles.css]: done.

**Comment by andreituicu** [comment]
I did this for blogs by simply adding a section with metadata via autoblocking: https://github.com/hlxsites/moleculardevices/blob/main/templates/blog/blog.js#L40-L50

in order to keep the CSS in just one place:
https://github.com/hlxsites/moleculardevices/blob/main/styles/styles.css#L753-L765

WDYT? I don't have anything against your approach, if you would like to keep it, it's just a suggestion if you like it better.

**Comment by lydiapuric** [comment]
@andreituicu Thank you. From modularity point I understand your view. However, I prefer simplicity: there is only 1 single css style for all the flavors for this curved footer and the auto-blocking part adds an abstraction and load on page I don't see the need of it. In case complexity might change, I would consider creating auto-blocking for it.

**Review by andreituicu — APPROVED** [review]

**Review by mhaack — APPROVED** [review]

**Pull request merged**